packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -1102,10 +1102,11 @@ abstract class ResidentRunner {
       'flutter',
       'png',
     );
-    final List<FlutterView> views = await device
-      .vmService.getFlutterViews();
+    List<FlutterView> views = <FlutterView>[];
     try {
       if (supportsServiceProtocol && isRunningDebug) {
```

Comment:
Optional cleanup: Pull the debug banner toggle out into a helper function.

Reply:
done